src/sentry/south_migrations/0501_auto__add_index_group_project_id.py

```diff
@@ -0,0 +1,1491 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = True
+
+    def forwards(self, orm):
+        # Adding index on 'Group', fields ['project', 'id'] to have
+        # project_id in the replication identity
+        index_name = db.create_index_name('sentry_groupedmessage', [u'project_id', 'id'])
+        if is_postgres():
```

Comment:
We don't need any of this `if is_postgres()` stuff anymore since we only support Postgres and no other database engine.

Reply:
ok